### DISCUSSION TITLE
Optimize the management of the color themes of the script editor and the Godot editor.

### DISCUSSION THREAD

**Discussion by PabloDevGit** [godotengine/godot-proposals#10313] [Editor]
### Describe the project you are working on

Not important

### Describe the problem or limitation you are having in your project

Changing colors to create a custom configuration for the script editor (and godot editor) can take you 3 hours due to the time it takes for godot to apply and load color configurations.
Simply try changing the background color of the script editor, 6 seconds later the new selected color will be applied.
Would it be possible to optimize this to make everything work a little faster?

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

Optimized the code that loads and applies color settings for godot editors.

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

...

### If this enhancement will not be used often, can it be worked around with a few lines of script?

No

### Is there a reason why this should be core and not an add-on in the asset library?

...

**Comment by AThousandShips** [comment]
Converted to a discussion as this doesn't contain any practical details or ideas for how to improve things
